### ISSUE TITLE
No tasks listed

### ISSUE THREAD

**Issue by HeikoAdams** [sgaraud/gnome-extension-taskwarrior#1]
After installing the extension no task are listed even if I've got 6 active tasks.
![bildschirmfoto von 2016-03-12 10-34-04](https://cloud.githubusercontent.com/assets/1418818/13722220/16bebf44-e83e-11e5-935c-68465a813877.png)

**Comment by sgaraud** [comment]
I have tested using up to version 2.4.4. Maybe the export format is different in your version.
could you check the output of the following command ?
task export status:pending

also do you get any output in "journalctl -f" when you click on the extension icon ?

**Comment by sgaraud** [comment]
I supposed that in the newer version of taskwarrior, the export command is returning by default a json array. https://taskwarrior.org/docs/commands/export.html
Could you try  putting "rc.json.array=off" in the "add filter" field ? If it works I will modify the code to set it for each export command.

**Comment by HeikoAdams** [comment]
After putting "rc.json.array=off" into the "add filter" field my tasks are listed :smile:

**Comment by sgaraud** [comment]
@HeikoAdams , great I will fix that asap to set the option explicitly in the export command.

**Comment by fdelehay** [comment]
I encountered the same problem recently despite being on Gnome 3.34 and with an up-to-date version. It had been working fine until now, but after clearing the list completely, I was unable to see newly added tasks. I reinstalled from the latest git version to be sure but no difference. Putting "rc.json.array=off" into the "add filter" field makes the task list display again. But any attempt to add a different filter makes the list disappear again.
